src/dags/overlastgebieden.py

```diff
@@ -1,53 +1,50 @@
 import operator
+from typing import Dict
+
 from airflow import DAG
 from airflow.models import Variable
 from airflow.operators.bash_operator import BashOperator
-from airflow.operators.postgres_operator import PostgresOperator
 from airflow.operators.dummy_operator import DummyOperator
-
-from contact_point.callbacks import get_contact_point_on_failure_callback
-from provenance_rename_operator import ProvenanceRenameOperator
-from postgres_rename_operator import PostgresTableRenameOperator
-from postgres_permissions_operator import PostgresPermissionsOperator
-from swift_operator import SwiftOperator
-
+from airflow.operators.postgres_operator import PostgresOperator
+from airflow.providers.sftp.operators.sftp import SFTPOperator
 from common import (
-    quote_string,
-    default_args,
-    pg_params,
-    slack_webhook_token,
     DATAPUNT_ENVIRONMENT,
     SHARED_DIR,
     MessageOperator,
+    default_args,
+    quote_string,
+    slack_webhook_token,
 )
+from common.db import DatabaseEngine
+from contact_point.callbacks import get_contact_point_on_failure_callback
+from ogr2ogr_operator import Ogr2OgrOperator
+from postgres_check_operator import COUNT_CHECK, GEO_CHECK, PostgresMultiCheckOperator
+from postgres_permissions_operator import PostgresPermissionsOperator
+from postgres_rename_operator import PostgresTableRenameOperator
+from provenance_rename_operator import ProvenanceRenameOperator
 
-from postgres_check_operator import (
-    PostgresMultiCheckOperator,
-    COUNT_CHECK,
-    GEO_CHECK,
-)
-
-dag_id = "overlastgebieden"
+dag_id: str = "overlastgebieden"
 
-variables_overlastgebieden = Variable.get("overlastgebieden", deserialize_json=True)
-files_to_download = variables_overlastgebieden["files_to_download"]
-tables_to_create = variables_overlastgebieden["tables_to_create"]
+variables_overlastgebieden: Dict = Variable.get("overlastgebieden", deserialize_json=True)
```

Comment:
Voor de `Dict` en `List` types nog de datatypes van de inhoud, of de key/value tovoegen.

Reply:
:+1: